### PR TITLE
i18n(zh-cn): Update translate from #6623

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update translate from #6623

Update files:
- [reference/error-reference.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/zh-cn/reference/error-reference.mdx)
- [reference/errors/astro-glob-no-match.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/zh-cn/reference/errors/astro-glob-no-match.mdx)

Create files:
- [reference/errors/i18n-not-enabled.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/zh-cn/reference/errors/i18n-not-enabled.mdx)
- [reference/errors/missing-index-for-internationalization.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/zh-cn/reference/errors/missing-index-for-internationalization.mdx)

#### Related issues & labels (optional)

- #6623

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/i18n-not-enabled.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: 未启用国际化
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/i18n-not-enabled.mdx:2]:
```suggestion
title: Index page not found.
```

File: src/content/docs/zh-cn/reference/errors/missing-index-for-internationalization.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: 未找到索引页面。
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+> **MissingIndexForInternationalization**: 无法找到索引页面。需要一个根索引页面，以便创建一个重定向到默认语言环境的索引 URL。(`/DEFAULT_LOCALE`)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/missing-index-for-internationalization.mdx:6]: index page 不要翻译成 索引页面

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**